test/Microsoft.ApplicationInsights.AspNetCore.Tests/Extensions/HttpRequestExtensionsTest.cs

```diff
@@ -99,7 +100,7 @@ public void TestGetUriReturnsCorrectUriIfRequestObjectSchemeAndHostAndPathAndQue
             var uri = HttpRequestExtensions.GetUri(request);
 
             Assert.Equal(
-                new Uri(string.Format("{0}://{1}{2}{3}", ExpectedSchema, ExpectedHostName, ExpectedPath, ExpectedQueryString)),
+                new Uri(string.Format(CultureInfo.InvariantCulture, "{0}://{1}{2}{3}", ExpectedSchema, ExpectedHostName, ExpectedPath, ExpectedQueryString)),
```

Comment:
I thought Ordinal does a basic string comparison, and invariant culture also understands different symbols for comparison, and is more strict. I don't think there will be an adverse effect on using invariant culture here. Please correct me if I am wrong!